src/languages/javascript.coffee

```diff
@@ -607,6 +615,29 @@ JavaScriptParser.drop = (block, context, pred) ->
 
   return helper.DISCOURAGE
 
+isStandardForLoop = (node) ->
+  unless node.init? and node.test? and node.update?
```

Comment:
Add a comment here giving an example of what's being looked for, similar to the readme.